### PR TITLE
Add some additional default keymaps

### PR DESCRIPTION
**Pull request by theutz**
I thought these might be a useful addition to the default keymaps as well:

- `to_title_case`
- `to_dot_case`
- `to_path_case`

Feel free to use this PR to add default keymaps for the other methods that are missing. I didn't want to start making too many additions that weren't mnemonic with the method name.

### CODE REVIEW

**Comment by johmsalas** [comment]
Hi @theutz, thanks for your contribution.

I didn't include them because those types are not commonly used specially as keymaps, they are actually useful for the `Subs` command

I'm quite curious about the usage you are making of those

**Comment by theutz** [comment]
Thanks for the feedback! I can see where those would definitely useful for the subs command, and I understand you not wanting to clutter up the keybindings too much.

My use case for the Title case command is that I'm often moving text back-and-forth between comments/docs and code, and often in the more user-facing text I'll refer to, let's say, lists of properties that are in snake case, but I want to list them in plain English.

I don't have as strong of a use case for dot and dash case, though. Perhaps those are excessive for the key bindings.

Let me know what you think!

**Comment by johmsalas** [comment]
Given this plugin scope is bigger than just code (ie: it supports unicode). I think title case is in fact a good candidate for the default list of keymaps.

File: lua/textcase/plugin/presets.lua [hunk 1]
```diff
@@ -46,6 +46,9 @@ local function setup_default_keymappings()
     { method_name = "to_pascal_case", quick_replace = "p", operator = "op", lsp_rename = "P" },
     { method_name = "to_upper_case", quick_replace = "u", operator = "ou", lsp_rename = "U" },
     { method_name = "to_lower_case", quick_replace = "l", operator = "ol", lsp_rename = "L" },
+    { method_name = "to_title_case", quick_replace = "t", operator = "ot", lsp_rename = "T" },
+    { method_name = "to_dot_case", quick_replace = "o", operator = "oo", lsp_rename = "O" },
```
Comment (johmsalas) [inline lua/textcase/plugin/presets.lua:50]: Could you comment these two lines of codes (mappings for to_dot_case and to_path_case) and add a comment on why they are not added as defaults?

I think such comment will be a guide for future contributors interested on the reason why they aren't added by default

Comment (theutz) [reply lua/textcase/plugin/presets.lua]: Sure! It's done.

**Review by johmsalas — APPROVED** [review]
Thanks!!

One last question about your use case, is Titile Case LSP rename something you use?
I'm considering removing the `lsp_rename = "T"` (I'd have to check if text-case supports partial key mappings)

**Comment by theutz** [comment]
You know, I don’t really use that, no. I just use the basic title case.

**Comment by Kayzels** [comment]
I see this pull request has been approved, but it hasn't been merged. Are there any plans to eventually merge it?
